e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -422,22 +422,22 @@ $iotHubName = az deployment group show -g $ResourceGroup -n $deploymentName --qu
 #################################################################################################################################################
 # Configure an AAD app and create self signed certs and get the bytes to generate more content info.
 #################################################################################################################################################
-Write-Host "`nCreating App Registration $logAnalyticsAppRegnName"
+Write-Host "`nCreating app registration $logAnalyticsAppRegnName"
 $logAnalyticsAppRegUrl = "http://$logAnalyticsAppRegnName"
-az ad sp create-for-rbac -n $logAnalyticsAppRegUrl --role "Reader" --scope $resourceGroupId --output none
-$logAnalyticsAppId = az ad app list --display-name $logAnalyticsAppRegnName --query "[?displayName=='$logAnalyticsAppRegnName'].appId" --output tsv
-Write-Host "`nApplication $logAnalyticsAppRegnName with Id $logAnalyticsAppId was created successfully."
+$logAnalyticsAppId = az ad sp create-for-rbac -n $logAnalyticsAppRegUrl --role "Reader" --scope $resourceGroupId --query "appId" --output tsv
```

Comment:
Get value from create, rather than listing after, avoiding timing issues where listing may not return the item.